open-sphere-base/icon-manager/src/main/java/io/opensphere/icon/manager/IconManagerFrame.java

```diff
@@ -0,0 +1,42 @@
+package io.opensphere.icon.manager;
+
+import java.io.FileInputStream;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+public class IconManagerFrame extends Application

```

Comment:
I assume this class will eventually go away?  If so lets put a little comment about this being a test driver or something, then we don't need to be as picky when reviewing changes for it.